src/getContext.ts

```diff
@@ -3,12 +3,12 @@ import {getInput} from '@actions/core'
 export type Context = ReturnType<typeof getContext>
 
 export const getContext = () => {
-  const token = getInput('github-token')
-  const repo = getInput('repo') || process.env.GITHUB_REPOSITORY!
+  const token = getInput('github-token') || process.env.GITHUB_TOKEN!
+  const repo = getInput('repo') || process.env.GITHUB_REPOSITORY || 'microsoft/TypeScript'
   const workspace = process.env.GITHUB_WORKSPACE!
-  const label = getInput('label')
-  const tag = getInput('code-tag')
-  const bisectIssue = getInput('bisect') as string | undefined
+  const label = getInput('label') || 'Has Repro'
+  const tag = getInput('code-tag') || 'repro'
+  const bisectIssue = getInput('bisect') || (process.env.BISECT_ISSUE as string | undefined)
```

Comment:
These changes make it easier to run locally, specifying everything you need as shell env variables.